daemon/algod/api/server/v2/account_test.go

```diff
@@ -62,38 +90,87 @@ func TestAccount(t *testing.T) {
 	require.Equal(t, conv.Address, addr)
 	require.Equal(t, conv.Amount, b.MicroAlgos.Raw)
 	require.Equal(t, conv.AmountWithoutPendingRewards, a.MicroAlgos.Raw)
+
 	require.NotNil(t, conv.CreatedApps)
-	require.Equal(t, 1, len(*conv.CreatedApps))
-	app := (*conv.CreatedApps)[0]
-	require.Equal(t, uint64(appIdx), app.Id)
-	require.Equal(t, params.ApprovalProgram, app.Params.ApprovalProgram)
-	require.Equal(t, params.GlobalStateSchema.NumUint, app.Params.GlobalStateSchema.NumUint)
-	require.Equal(t, params.GlobalStateSchema.NumByteSlice, app.Params.GlobalStateSchema.NumByteSlice)
-	require.NotNil(t, conv.AppsLocalState)
-	require.Equal(t, 1, len(*conv.AppsLocalState))
+	require.Equal(t, 2, len(*conv.CreatedApps))
+	for _, app := range *conv.CreatedApps {
```

Comment:
the loop is because order in resultant `CreatedApps` is not fixed due to random order of maps iteration